package exceptionhandling;

public class TryCatchFinally {

	public static void main(String[] args) {
		String a="abc";
		try
		{
			System.out.println(a.charAt(10));
		} 
		catch (NullPointerException e) 
		{
			System.out.println("check index");
			
		}
		finally {
			System.out.println("finally block is running");
		}
		

	}

}
